crankshaft-docker/CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#8](https://github.com/stjude-rust-labs/crankshaft/pull/8)).
 * Replaced `attached` with separate stdout and stderr attach flags
   ([#8](https://github.com/stjude-rust-labs/crankshaft/pull/8)).
+* Updated `bollard` to an official, upstreamed crate release
```

Comment:
FWIW, I consider the change to `crankshaft-engine` to be so small that it doesn't need to be included in the `CHANGELOG.md`.